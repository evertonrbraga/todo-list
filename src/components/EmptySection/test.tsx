import { render, screen } from 'project-testing-library'
import { EmptySection } from '.'

describe('<EmptySection />', () => {
  it('should snapshot the EmptySection component', () => {
    const { container } = render(<EmptySection />)
    expect(container.firstChild).toMatchSnapshot()
  })

  it('should if the title is rendering correctly', () => {
    render(<EmptySection />)
    const title = screen.getByText('Você ainda não tem tarefas cadastradas')
    expect(title).toBeInTheDocument()
  })
})
