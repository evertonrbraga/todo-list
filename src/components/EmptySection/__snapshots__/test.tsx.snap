// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptySection /> should snapshot the EmptySection component 1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin-top: 50px;
  color: #808080;
}

.c0 img {
  margin-bottom: 20px;
}

.c0 h1 {
  margin-bottom: 2px;
  font-size: 1.2rem;
}

<main
  class="c0"
>
  <img
    alt="Ícone prancheta"
    src="test-file-stub"
  />
  <h1>
    Você ainda não tem tarefas cadastradas
  </h1>
  <p>
    Crie tarefas e organize seus itens a fazer
  </p>
</main>
`;
